pilot/pkg/config/clusterregistry/conversion.go

```diff
@@ -61,8 +61,8 @@ func (cs *ClusterStore) GetClusterAccessConfig(cluster *k8s_cr.Cluster) *clientc
 	return &clusterAccessConfig
 }
 
-// GetClusterName returns a cluster's name
-func GetClusterName(cluster *k8s_cr.Cluster) string {
+// GetClusterID returns a cluster's ID
+func GetClusterID(cluster *k8s_cr.Cluster) string {
```

Comment:
Generally best to avoid renaming methods when the change is mostly cosmetic...

Reply:
The reason for this change is that cluster name has been widely used to identify envoy cluster.